bazel/envoy_build_system.bzl

```diff
@@ -0,0 +1,39 @@
+# References to Envoy external dependencies should be wrapped with this function.
+def envoy_external_dep_path(dep):
+    return "//external:%s" % dep
```

Comment:
It doesn't really help. I'm just saying that adding this indirection and accessing targets via `//external:googletest_main` doesn't provide any benefits over accessing it directly as `@googletest//:googletes_main`, and the fact that you see `//external` in other places is mostly due to historic reasons.

Please note that it might be just a matter of taste, at this point.